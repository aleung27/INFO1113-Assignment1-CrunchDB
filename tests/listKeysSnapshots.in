LIST KEYS
SET a 8
SNAPSHOT
SET B 9
SNAPSHOT
LIST KEYS
CHECKOUT 1
LIST KEYS
CHECKOUT 2
LIST KEYS
SET C 0
LIST KEYS
ROLLBACK 1
LIST KEYS
