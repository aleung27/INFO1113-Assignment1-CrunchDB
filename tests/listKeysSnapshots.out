> no keys

> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> B
a

> ok

> a

> ok

> B
a

> ok

> C
B
a

> ok

> a

